### ISSUE TITLE
Rename adult dependent CTC to ODC

### ISSUE THREAD

**Issue by MaxGhenis** [PolicyEngine/policyengine-us#1585]
The IRS refers to this as the ODC. @nikhilwoodruff did you name it adult dependent CTC because it's in the same part of the IRC as the CTC?

**Comment by nikhilwoodruff** [comment]
Yes, the IRC refers to it as an increase to the CTC ("the credit allowed under a)")

<img width="837" alt="image" src="https://user-images.githubusercontent.com/35577657/208649388-ce21bda4-744c-4ec2-9985-8ca1d4655d56.png">
